.github/workflows/scripts/post_before_script.sh

```diff
@@ -22,3 +22,8 @@ cmd_prefix chmod a+x /var/lib/pulp/sign-metadata.sh
 KEY_FINGERPRINT="6EDF301256480B9B801EBA3D05A5E6DA269D9D98"
 TRUST_LEVEL="6"
 echo "$KEY_FINGERPRINT:$TRUST_LEVEL:" | cmd_stdin_prefix gpg --import-ownertrust
+
+
+if [[ "$GITHUB_EVENT_NAME" == "schedule" ]]; then
```

Comment:
Apologies Fabricio it seems I always have another way to suggest today.

I'd like us to try to do this differently. I want the plugin template to have it's nightly job run tests marked with the `nightly` pytest mark and have the functional tests leave out the `nightly` ones.

Reply:
What do you think?

Reply:
sounds good

Reply:
Is making this something you want to try or should I? I'm keep asking because I want to get more people involved in this transition.